nix/sources.json

```
@@ -36,27 +36,27 @@
         "url_template": "https://github.com/<owner>/<repo>/archive/<rev>.tar.gz"
     },
     "nixpkgs": {
-        "branch": "release-24.05",
+        "branch": "release-25.05",
```

https://github.com/NixOS/nixpkgs/commit/c46290747b2aaf090f48a478270feb858837bf11
```suggestion
        "branch": "c462907",
```